src/styles/blocks/card.scss

```diff
@@ -0,0 +1,122 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+$color-card-blue: #00acdc;
+$color-card-dark:#060b35;
+
+html {
+  font-family: "Roboto", sans-serif ;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
```

Comment:
it should be only bem block styles here

create `reset.scss` and move these styles there